ext/cfx-ui/src/app/servers/connecting-popup.component.ts

```diff
@@ -19,6 +19,7 @@ export class ConnectingPopupComponent implements OnInit {
 	closeLabel = "#Servers_CloseOverlay";
 	retryLabel = "#Servers_Retry";
 	submitting = false;
+	closeKeys = [27, 8, 46];
```

Comment:
Magic numbers - everyone's favorite! `event.key` or `event.code` might help with readability as it'd not involve uncommented numbers?